BEP92.md

```diff
@@ -0,0 +1,51 @@
+# BEP-92: Implementation of SHA3-256 FIPS 202 hash and EcRecover Uncompressed public key precompiled contracts
+
+- [BEP-92: Implementation of SHA3-256 FIPS 202 hash precompiled and EcRecover Uncompressed public key precompiled](#bep-92)
+  - [1. Summary](#1-summary)
+  - [2. Abstract](#2-abstract)
+  - [3. Status](#3-status)
+  - [4. Motivation](#4-motivation)
+  - [5. Specification](#5-specification)
+  - [6. License](#6-license)
+  
+## 1. Summary
+
+This BEP describes a proposal for Implementation of SHA3-256 as precompiled for BSC
+
+## 2. Abstract
+
+BEP-92 Proposal describes a addition of implementation of SHA3-256 FIPS 202 hash and EcRecover Uncompressed public key methods as precompiled native contracts
+
+## 3. Status
+
+This BEP is a Work in Progress(WIP). 
+
+## 4. Motivation
+
+To bring more interoperability to Binance Smart Chain, this BEP proposing to add additional capabilities to verify sha3-256 based blockchains in the likes of ICON republic.
+
+- In order to validate the merkle tree data from ICON and other similar blockchains, which uses the SHA3-256 hashing function. 

```

Comment:
Please use `ICON` as less as possible and make the bep more generic. Thanks.

Reply:
@guagualvcha , Sure, I'll modify this.

Reply:
thanks. LGTM